### ISSUE TITLE
[Cheetah MVP] Foundational accessibility testing checklist

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#21342]
## Scope
All pages in Cheetah flow
- /health-care/schedule-view-va-appointments/appointments/
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-facilities
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/received-dose
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-facilities
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/facility
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/clinic
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/select-date-1
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/plan-second-dose
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-info
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/review
- /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/confirmation

## Tasks
- Document any issues found in separate tickets

## Acceptance Criteria
Complete required accessibility checklist items for staging review and check off items as they are completed:
  - [ ] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#accessibility-staging-review-preparation)
  - [ ] [Axe scans in daily development](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-daily-development)
    - Conduct axe browser scan, and No violations indicated in axe scan results OR Axe scan results in violations, but those violations are documented in issue ticket(s)
  - [ ] [Axe scans in end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-end-to-end-tests)
    - No violations indicated in end-to-end tests OR End-to-end tests result in violations, but those violations are documented in issue ticket(s)
  - [ ] [Color tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#color-tests)  
    - Check color contrast
    - Check for colorblindness issues
  - [ ] [Content resize check](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#content-resize-check)
    - Zoom to 200% at 1280px wide and confirm readability, no info cut off, etc
    - Repeat for 300%
    - Repeat for 400%
  - [ ] [Keyboard navigation check](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#keyboard-navigation-check)
    - Navigate all pages and core functionality with keyboard only
  - [ ] [Screen reader tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#screen-reader-tests)
    - Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox

**Comment by grossb1** [comment]
Whoever picks up this ticket, I'd like to pair on this!

**Comment by jenniemc** [comment]
@laurenernest, how do you want to have accessibility issues annotated when going through the checklist? Do you want to have issues listed in this ticket?  cc @grossb1

**Comment by laurenernest** [comment]
Write up individual tickets, like #20725 or #20401 for example.
